.java-version

```diff
@@ -0,0 +1 @@
+1.7
```

Comment:
what is this file? I'm wondering if it's something that should be added to `.gitignore` instead of committed